<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>370</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Temperature Converter</string>
  </property>
  <widget class="QGroupBox" name="groupBoxK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>220</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>Kelvin</string>
   </property>
   <widget class="QDial" name="dialK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>4000</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchTarget">
     <double>10.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
   <zorder>lcdNumberK</zorder>
   <zorder>dialK</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBoxC">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>220</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>Celsius</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLCDNumber" name="lcdNumberC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
   <widget class="QDial" name="dialC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-273</number>
    </property>
    <property name="maximum">
     <number>3727</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchTarget">
     <double>10.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxF">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>220</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>Fahrenheit</string>
   </property>
   <widget class="QDial" name="dialF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-460</number>
    </property>
    <property name="maximum">
     <number>6866</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchTarget">
     <double>10.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
   <zorder>lcdNumberF</zorder>
   <zorder>dialF</zorder>
  </widget>
  <zorder>groupBoxK</zorder>
  <zorder>groupBoxF</zorder>
  <zorder>groupBoxC</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialF</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberF</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialK</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberK</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberC</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
